cypress/utils/index.d.ts

```diff
@@ -3,14 +3,119 @@
 
 declare namespace Cypress {
   interface Chainable<Subject> {
+    /**
```

Comment:
I don't have much context on this file, but looks like most of the customized commands are not added in this file? I see there are three ways to add customized commands:
* just add a command in cypress/utils/plugins/{plugin}/commands.js
* add a command in cypress/utils/plugins/{plugin}/commands.js and then declare the command in cypress/utils/plugins/{plugin}/index.d.ts, example: https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/utils/plugins/security-dashboards-plugin/index.d.ts
* add a command in cypress/utils/plugins/{plugin}/commands.js and then declare the command in cypress/utils/index.d.ts

Do we have a preference? @kavilla @ashwin-pc @ohltyler

Reply:
@CCongWang I have refactored commands, all are now in the commands file and declared in index.d.ts within the plugins folder.